### PR TITLE
feat(dataset): tornadoes-2024-dow-refl-greenfield

### PR DESCRIPTION
**Pull request by anayeaye**
## What
Add stac collection and ingestion config for `tornadoes-2024-dow-refl-greenfield`

## Parent issue
https://github.com/NASA-IMPACT/veda-data/issues/187

## Downstream veda-config PR
https://github.com/NASA-IMPACT/veda-config/pull/420

## Checklist
- [x] staging config completed and verified
- [x] published to staging
- [x] transfer config and production metadata updated
- [x] published to production

### CODE REVIEW

File: ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    ],
+    "renders": {
+        "dashboard": {
+            "bidx": [1],
+            "colormap_name": "turbo",
+            "nodata": -32768,
+            "assets": [
+                "cog_default"
+            ],
+            "rescale": [-10, 50],
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json:64]:
```suggestion
            "rescale": [[-10, 50]],
```

File: ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            "roles": [
+                "host"
+            ]
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Thumbnail",
+            "description": "Photo by [Jonny Glessner](https://x.com/JonnyGlessner/status/1768424574855610777/photo/4) (Wedge tornado passing southeast of Wapakoneta, Ohio on March 14, 2024)",
+            "href": "https://thumbnails.openveda.cloud/tornado-2024-cover.png",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json:34]:
```suggestion
            "description": "Photo by [Ryan McGinnis](https://static.wikia.nocookie.net/stormchasers/images/1/1b/DOW.jpg/revision/latest?cb=20100115235828) (DOW6 parked in front of a tornadic mesocyclone in the Plains)",
            "href": "https://thumbnails.openveda.cloud/tornadoes-2024-dow-background.jpg",
```

**Comment by anayeaye** [comment]
The asset for this collection needed to be cloud optimized. I created a v2 of the file

`gdal_translate -of COG -co COMPRESS=DEFLATE /vsis3/veda-data-store-staging/2024tornadoes/DOW_Greenfield_REFL_cog_2024-05-21.tif s3://veda-data-store-staging/2024tornadoes/DOW_Greenfield_REFL_cog_v2_2024-05-21.tif`

`aws s3 cp temp/DOW_Greenfield_REFL_cog_v2_2024-05-21.tif s3://veda-data-store-staging/2024tornadoes/DOW_Greenfield_REFL_cog_v2_2024-05-21.tif`

File: ingestion-data/production/dataset-config/tornadoes-2024-dow-refl-greenfield.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
+{
+    "collection": "tornadoes-2024-dow-refl-greenfield",
+    "title": "DOW7 Reflectivity (Greenfield, IA)",
+    "description": "Horizontal Reflectivity values (dBZ) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
```
Comment (botanical) [inline ingestion-data/production/dataset-config/tornadoes-2024-dow-refl-greenfield.json:4]:
```suggestion
    "description": "Horizontal reflectivity values (dBZ) from the EF-4 Greenfield, IA tornado on May 21, 2024 collected by DOW7. Scan taken at 20:41:08.",
```

I don't know how important this is but this is what veda-config has

File: ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
+{
+    "collection": "tornadoes-2024-dow-refl-greenfield",
+    "title": "DOW7 Reflectivity (Greenfield, IA)",
+    "description": "Horizontal Reflectivity values (dBZ) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json:4]:
```suggestion
    "description": "Horizontal reflectivity values (dBZ) from the EF-4 Greenfield, IA tornado on May 21, 2024 collected by DOW7. Scan taken at 20:41:08.",
```

File: ingestion-data/production/dataset-config/tornadoes-2024-dow-refl-greenfield.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            "roles": [
+                "host"
+            ]
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Thumbnail",
+            "description": "Photo by [Jonny Glessner](https://x.com/JonnyGlessner/status/1768424574855610777/photo/4) (Wedge tornado passing southeast of Wapakoneta, Ohio on March 14, 2024)",
+            "href": "https://thumbnails.openveda.cloud/tornado-2024-cover.png",
```
Comment (botanical) [inline ingestion-data/production/dataset-config/tornadoes-2024-dow-refl-greenfield.json:34]:
```suggestion
            "description": "Photo by [Ryan McGinnis](https://static.wikia.nocookie.net/stormchasers/images/1/1b/DOW.jpg/revision/latest?cb=20100115235828) (DOW6 parked in front of a tornadic mesocyclone in the Plains)",
            "href": "https://thumbnails.openveda.cloud/tornadoes-2024-dow-background.jpg",
```

File: ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+            "roles": [
+                "host"
+            ]
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Thumbnail",
+            "description": "Photo by [Ryan McGinnis](https://static.wikia.nocookie.net/stormchasers/images/1/1b/DOW.jpg/revision/latest?cb=20100115235828) (DOW6 parked in front of a tornadic mesocyclone in the Plains)",
+            "href": "https://thumbnails.openveda.cloud/tornadoes-2024-dow-background.jpg",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json:34]:
```suggestion
            "description": "Photo by [Ryan McGinnis](https://static.wikia.nocookie.net/stormchasers/images/1/1b/DOW.jpg/revision/latest?cb=20100115235828) (DOW6 parked in front of a tornadic mesocyclone in the Plains)",
            "href": "https://thumbnails.openveda.cloud/tornadoes-2024-dow-background.jpg",
```

**Review by botanical — CHANGES_REQUESTED** [review]
This needs the new thumbnail

File: ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json (hunk 6 above)
Comment (anayeaye) [reply ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json:34]: This one got fixed outside of the PR so I am resolving

**Comment by anayeaye** [comment]
Thanks for catching the inconsistencies on this @botanical! I've updated the collection in the production catalog with the fixes

**Review by botanical — APPROVED** [review]
🎉

**Pull request merged**